src/components/TableWithControlsLayout/TableWithControlsLayout.scss

```diff
@@ -20,7 +20,7 @@
     &__controls {
         z-index: 3;
 
-        width: 100%;
+        width: max-content;
```

Comment:
`width: 100%` here is for the create db button - it should stick to the right

Expected:
<img width="1565" alt="Screenshot 2025-05-21 at 15 46 53" src="https://github.com/user-attachments/assets/08c93b4b-0ead-4cd5-8b25-21b1909a0aa8" />


You code:
<img width="1574" alt="Screenshot 2025-05-21 at 15 45 11" src="https://github.com/user-attachments/assets/67d35edd-5bfa-4c19-8fc0-9b1f9ec1c010" />